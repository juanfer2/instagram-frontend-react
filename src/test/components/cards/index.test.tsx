import React from 'react'
import { shallow } from 'enzyme'

import Card from '../../../components/cards'

describe('Pruebas en <Card />', () => {
  test('must be render component <AvatarCircle />', () => {
    const message = 'Card Component'
    const children = <p>{message}</p>
    const wrapper = shallow(<Card> {children} </Card>)
    expect(wrapper).toMatchSnapshot()
  })

  test('must be component <Card />', () => {
    const message = 'Card Component'
    const children = <p>{message}</p>
    const wrapper = shallow(<Card> {children} </Card>)
    const validateChildren = wrapper.find('p').text()

    expect(message).toBe(validateChildren)
  })
})
