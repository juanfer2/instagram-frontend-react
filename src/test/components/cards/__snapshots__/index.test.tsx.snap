// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <Card /> must be render component <AvatarCircle /> 1`] = `
<div
  className="card-notifications"
>
   
  <p>
    Card Component
  </p>
   
</div>
`;
